src/frontend/js/widgets/Dashboard/utils/teacherRouteMessages.tsx

```diff
@@ -0,0 +1,158 @@
+    description: 'Label of the teacher dashboard root view.',
+    defaultMessage: 'My teacher dashboard',
+  },
+  [TeacherDashboardPaths.TEACHER_PROFILE]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.profile.label',
+    description: 'Label of the teacher profile view.',
+    defaultMessage: 'My profile',
+  },
+  [TeacherDashboardPaths.TEACHER_SETTINGS]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.profile.settings.label',
+    description: 'Label of the teacher profile settings view.',
+    defaultMessage: 'Settings',
+  },
+  [TeacherDashboardPaths.TEACHER_COURSES]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.profile.courses.label',
+    description: 'Label of the teacher courses liste view.',
+    defaultMessage: 'Manage my courses',
+  },
+  [TeacherDashboardPaths.TEACHER_NOTIFICATIONS]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.profile.notifications.label',
+    description: 'Label of the teacher notifications view.',
+    defaultMessage: 'My notifications',
+  },
+  [TeacherDashboardPaths.ORGANIZATION]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.organization.label',
+    description: 'Label of the organization view.',
+    defaultMessage: 'Organization settings',
+  },
+  [TeacherDashboardPaths.ORGANIZATION_SETTINGS]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.organization.settings.label',
+    description: 'Label of the organization settings view.',
+    defaultMessage: 'Organization settings',
+  },
+  [TeacherDashboardPaths.ORGANIZATION_COURSES]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.organization.courses.label',
+    description: 'Label of the organization courses view.',
+    defaultMessage: 'Organization courses',
+  },
+  [TeacherDashboardPaths.ORGANIZATION_MEMBERS]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.organization.members.label',
+    description: 'Label of the unisersity members view.',
+    defaultMessage: 'Organization members',
+  },
+  [TeacherDashboardPaths.COURSE]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.course.label',
+    description: 'Label of the course view.',
+    defaultMessage: 'Course settings',
+  },
+  [TeacherDashboardPaths.COURSE_SETTINGS]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.course.settings.label',
+    description: 'Label of the course settings view.',
+    defaultMessage: 'Course settings',
+  },
+  [TeacherDashboardPaths.COURSE_CLASSROOMS]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.course.classroom.label',
+    description: 'Label of the course classroom view.',
+    defaultMessage: 'Course classroom',
+  },
+  [TeacherDashboardPaths.COURSE_RECORDS]: {
+    id: 'components.TeacherDashboard.TeacherDashboardRoutes.course.records.label',
+    description: "Label of the course's records view.",
+    defaultMessage: "Course's records",
+  },
```

Comment:
I think you should remove `Organization` that prepend all organization menu entries.
Same for course.